kong/db/migrations/core/003_1.0.0_to_1.0.1.lua

```diff
@@ -0,0 +1,23 @@
+return {
+  postgres = {
+    up = [[
+
+      UPDATE consumers SET created_at = DATE_TRUNC('seconds', created_at);
+      UPDATE plugins   SET created_at = DATE_TRUNC('seconds', created_at);
+      UPDATE upstreams SET created_at = DATE_TRUNC('seconds', created_at);
+      UPDATE targets   SET created_at = DATE_TRUNC('milliseconds', created_at);
+
+    ]],
+
+    teardown = function(connector, helpers)
+    end,
```

Comment:
Just as a general guideline for the future: there would be no need to specify the `teardown` functions if those are empty. It's an optional attribute of the migrations schema.